handlers/ec2/resources/scripts/nova-essex-start

```diff
@@ -67,13 +67,7 @@ try:
                              os.environ['EC2_PUBLIC_NETWORK'],
                              os.environ['PROJECT_NAME'],
                              os.environ['USER_NAME'],
-                             os.environ['EC2_SLICE_USER_EMAIL'],
```

Comment:
Same here - not needed?

Reply:
No, not needed as they are being passed only for project creation in create.project macro. No longer needed for vm start